### PR TITLE
[bulk] Validation: headings empty, disallowed HTML

### PR DESCRIPTION
**Pull request by LSchaufMS**
[bulk] Validation: headings empty (4), disallowed HTML, duplicate titles, missing descriptions, duplicate H1s.

This PR fixes 28 files.

### CODE REVIEW

File: Skype/SDN/articles/echopercentmicin-element-qualitypropertiestype-complextype-skype-sdn-2-2-d.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿---
-title: EchoPercentMicIn element (QualityPropertiesType complexType) 
+title: EchoPercentMicIn element, QualityPropertiesType complexType
 TOCTitle: EchoPercentMicIn element
+description: Information about the EchoPercentMicIn element.
```
Comment (Roudabeh1348) [inline Skype/SDN/articles/echopercentmicin-element-qualitypropertiestype-complextype-skype-sdn-2-2-d.md:4]: word count falls below the threshold, but feel free to take the comment with a grain of salt. :)

File: Skype/SDN/articles/useragent-element-inviteendpointtype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿---
-title: UserAgent element (InviteEndPointType complexType) 
+title: UserAgent element, InviteEndPointType complexType, schema D 
 TOCTitle: UserAgent element (InviteEndPointType complexType)
+description: Element information for the UserAgent element.
```
Comment (Roudabeh1348) [inline Skype/SDN/articles/useragent-element-inviteendpointtype-complextype-skype-for-business-sdn-interface-2-2-schema-d.md:4]: same thing for this one, word count. :)

File: Skype/SDN/articles/msdiagnostics-element-endedproperties-complextype-skype-for-business-sdn-interface-2-2-schema-c.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ﻿---
-title: MSDiagnostics element (EndedProperties complexType) 
+title: MSDiagnostics element, EndedProperties complexType, schema C
 TOCTitle: MSDiagnostics element (EndedProperties complexType)
+description: Additional information related to an error.
```
[the remark this answers is not in the document]
Comment (LSchaufMS) [reply Skype/SDN/articles/msdiagnostics-element-endedproperties-complextype-skype-for-business-sdn-interface-2-2-schema-c.md:4]: For these pages, I don't know that 100 characters is possible, unless we pad the heck out of the descriptions. They're very brief, content-light pages.

**Comment by LSchaufMS** [comment]
**Global effort to fix build validation errors**

@lindalu-MSFT 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Pull request merged**